File: fix.py
import glob
for f in glob.glob('Frontend/*.html'):
    with open(f, 'r', encoding='utf-8') as file:
        content = file.read()
    content = content.replace('<link rel=\"stylesheet\" href=\"css/style.css\">', '<link rel=\"stylesheet\" href=\"css/style.css\">\n    <link rel=\"stylesheet\" href=\"css/responsive.css\">')
    with open(f, 'w', encoding='utf-8') as file:
        file.write(content)

File: fix.js
const fs = require('fs');
const path = require('path');
const files = fs.readdirSync('Frontend').filter(f => f.endsWith('.html')).map(f => path.join('Frontend', f));
files.forEach(f => {
    let content = fs.readFileSync(f, 'utf8');
    content = content.replace('<link rel=\"stylesheet\" href=\"css/style.css\">', '<link rel=\"stylesheet\" href=\"css/style.css\">\\n    <link rel=\"stylesheet\" href=\"css/responsive.css\">');
    fs.writeFileSync(f, content, 'utf8');
});
